packages/cli/src/build/helpers/transpile.ts

```diff
@@ -22,7 +22,7 @@ export const transformImports = (target: Target, options: MitosisConfig) => (cod
       `${getFileExtensionForTarget({ type: 'import', target, options })}$1`,
     )
     .replace(
-      `.${options.extension}`,
+      new RegExp(`.${options.extension}`, 'g'),
```

Comment:
I realize now we probably wanna make sure it's at least followed by quotes, to avoid replacing `.svelte` strings in random parts of the code:

```suggestion
      new RegExp(`.${options.extension}['"]`, 'g'),
```